framework/src/play/libs/Files.java

```diff
@@ -31,7 +31,7 @@
      * @param to
      */
     public static void copy(File from, File to) {
-        if (from.getAbsolutePath().equals(to.getAbsolutePath())) {
+        if (0==from.toPath().compareTo(to.toPath())) {
```

Comment:
How can you be sure that getPath will compare the same file: ex with 
        File a = new File("C:\\temp\\TEST.TXT");
        File b = new File("C:\\temp\\.\\test.txt");
File should be detect as the are the same...